src/msw/toplevel.cpp

```diff
@@ -192,7 +192,7 @@ WXDWORD wxTopLevelWindowMSW::MSWGetStyle(long style, WXDWORD *exflags) const
             //    the taskbar normally)
             //
             // The second one is solved here by using WS_EX_APPWINDOW flag, the
-            // first one is dealt with in our MSWGetParent() method
+            // first one is dealt within our MSWGetParent() method
```

Comment:
This one is definitely wrong, the meaning is that it is "dealt with" "in ... method". Maybe the current comment awkward but the new one is just wrong.

Reply:
I see.  Yes, "with in" is correct.